src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String color;
+    private String figureName;
+
+    public String getColor() {
+        return color;
+    }
+
+    public String getFigureName() {
+        return figureName;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setFigureName(String figureName) {
+        this.figureName = figureName;
+    }
+
+    public abstract String drawFigure();
```

Comment:
Let's move this method to a separate interface

Reply:
done